easybuild/easyconfigs/m/METIS/METIS-5.1.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,34 @@
+name = 'METIS'
+version = '5.1.0'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/metis/overview'
+
+description = """
+ METIS is a set of serial programs for partitioning graphs, partitioning
+ finite element meshes, and producing fill reducing orderings for sparse
+ matrices. The algorithms implemented in METIS are based on the multilevel
+ recursive-bisection, multilevel k-way, and multi-constraint partitioning
+ schemes.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD',
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['76faebe03f6c963127dbb73c13eab58c9a3faeae48779f049066a21c087c5db2']
+
+# We use 32bit for indices and 64bit for content
+patches = ['METIS-5.1.0-use-doubles.patch']
```

Comment:
@JackPerdue One tiny remark: please don't hardcode the version here, use the `%(version)s` template:

```python
patches = ['METIS-%(version)s-use-doubles.patch']